lib/model/folder_sendrecv.go

```diff
@@ -1092,6 +1092,20 @@ func (f *sendReceiveFolder) renameFile(cur, source, target protocol.FileInfo, sn
 // handleFile queues the copies and pulls as necessary for a single new or
 // changed file.
 func (f *sendReceiveFolder) handleFile(file protocol.FileInfo, snap *db.Snapshot, copyChan chan<- copyBlocksState) {
+	
+	available := false
+   	for _, dev := range snap.Availability(file.Name) {
```

Comment:
Just cosmetics, but please fix the current mixture of spaces and tabs that you've got here. All lines in this file are affected. The spaces should be changed into tabs.

Reply:
Thanks Tomasz, no idea how I managed that!